gallery/src/components/NxForm/NxFormPage.tsx

```diff
@@ -230,6 +230,17 @@ const NxFormPage = () =>
               function in order to avoid rendering children before loading completes
             </NxTable.Cell>
           </NxTable.Row>
+          <NxTable.Row>
```

Comment:
What's this about?  I wouldn't expect a new prop on `NxForm` in this PR, and this one isn't described in a way that makes it sound like it goes on `NxForm` at all

Reply:
Reviewing the documentation on the page, I misunderstood and thought I needed to add the documentation for isRequired. It has been removed.

Reply:
Reviewing the documentation on the page, I misunderstood and thought I needed to add the documentation for isRequired. It has been removed.

Reply:
> What's this about? I wouldn't expect a new prop on `NxForm` in this PR, and this one isn't described in a way that makes it sound like it goes on `NxForm` at all

Reviewing the documentation on the page, I misunderstood and thought I needed to add the documentation for isRequired. It has been removed.

Reply:
> The row that this thread is about is still present. That commit only affects other files.

Sorry, now removed here: [97b2c40](https://github.com/sonatype/sonatype-react-shared-components/pull/735/commits/97b2c4096f827cb510a729cd2317fa689a60ab5b)